### DISCUSSION TITLE
Deep scan not working

### DISCUSSION THREAD

**Discussion by praveenbakare** [tokern/piicatcher#243] [General]
I am trying to run piicatcher 0.21.2 on Bigquery tables with deep scan. But the results are showing null pii-type. When I run the detect with shallow scan, I get the results where pii_type is populated for some of the columns. What do I need to do to make sure that Deep scan also works and provides results ?

Deep Scan: piicatcher detect --source-name bq_db_qa --scan-type data --list-all --no-incremental
Shallow Scan: piicatcher detect --source-name bq_db_qa --scan-type metadata --list-all --no-incremental

**Comment by vrajat** [comment]
First suspicion is that piicatcher is not able to read the data. No rows are being returned. Can you share debug logs?

**Reply by praveenbakare** [reply]
I would not be able to share logs as it is executed in our company restricted environment. But is there anything in particular that I can look for in the logs to see (or print) in order to check if data is not read? I can add tht in piicatcher code and execute it.
